test/functional/apps/visualize/_chart_types.ts

```diff
@@ -37,9 +33,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     it('should show the correct agg based chart types', async function () {
       await PageObjects.visualize.clickAggBasedVisualizations();
-      let expectedChartTypes = [
+      const expectedChartTypes = [
         'Area',
-        'Coordinate Map',
```

Comment:
Does this mean we won't test whether coordinate map and region map show up at all?

Reply:
On this test yes but we already test it [here](https://github.com/elastic/kibana/blob/master/test/functional/apps/visualize/_region_map.ts) 
Not directly, but this test opens the wizard and clicks on the region map. Unfortunately, when the ci stops running the tests for OSS the above test won't also run. 
`if (isOss) {
        loadTestFile(require.resolve('./_tile_map'));
        loadTestFile(require.resolve('./_region_map'));
      }`
We have to discuss this with the maps team.

Reply:
Got it, makes sense